imkar/analytical/analytical.py

```diff
@@ -0,0 +1,169 @@
+import numpy as np
+import pyfar as pf
+import numbers
+
+def rectangular(frequency_vector, phi_vector, width, length, height_to_length):
```

Comment:
```suggestion
def rectangular(frequencies, phi_vector, width, length, hight, c=343):
```
better to just give absolute inputs and not relative ones
usually we call sth like ``frequency_vector`` ``frequencies``, so better to rename it.

Reply:
frequency_vector, phi_vector and height_to_length parameter changed. c as additional parameter added.